requirements-dev.in

```diff
@@ -1,6 +1,6 @@
 -c requirements.txt
 
-awscli~=1.34.29
+awscli>=1.35
```

Comment:
Why the changes from `~=` to `>=`?

Reply:
other dependencies were upped...impacting these libs